examples/gke/cluster.yaml

```diff
@@ -82,7 +82,7 @@ metadata:
   namespace: scylla
 spec:
   version: 4.0.0
-  agentVersion: 2.0.2
+  agentVersion: 2.2
```

Comment:
There is no `2.2` image in docker hub, but there is a `2.2.0`.
https://hub.docker.com/r/scylladb/scylla-manager-agent/tags